package.json

```diff
@@ -16,28 +16,24 @@
   },
   "author": "Josh Sommer <joshdsommer@gmail.com> (https://twitter.com/_JoshSommer)",
   "license": "ISC",
-  "dependencies": {},
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.8.0",
-    "@angular/cli": "6.x",
-    "@angular/common": "6.x",
-    "@angular/compiler": "6.x",
-    "@angular/compiler-cli": "6.x",
-    "@angular/core": "6.x",
-    "@angular/forms": "6.x",
-    "@angular/http": "6.x",
-    "@angular/platform-browser": "6.x",
-    "@angular/platform-browser-dynamic": "6.x",
+    "@angular-devkit/build-angular": "~0.13.0",
+    "@angular/cli": "7.3.10",
+    "@angular/common": "7.2.16",
+    "@angular/compiler": "7.2.16",
+    "@angular/compiler-cli": "7.2.16",
+    "@angular/core": "7.2.16",
+    "@angular/forms": "7.2.16",
+    "@angular/http": "7.2.16",
+    "@angular/platform-browser": "7.2.16",
+    "@angular/platform-browser-dynamic": "7.2.16",
```

Comment:
It looks like the `x` in the version number was added when `ng-packagr` was installed. I'm not quite sure what the `x`'s are. Are they important?

Reply:
The x's in version numbers simply mean that it will grab the latest sub-version within that major version.  Adding dedicated version numbers are fine too.  Especially since this is on the first of many updates.

Reply:
Yeah, that sounds right. Basically sounds like an alternative to `~` syntax, from what the npm docs say.

https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept

Reply:
updated to .x syntax